### PR TITLE
platforms/azure: add cloudapp dns name

### PR DESCRIPTION
**Pull request by sozercan**
Adds Azure <resourceGroupName>.cloudapp.azure.com FQDN if `tectonic_create_dns_zone = "false"` in tfvars which makes Azure DNS Zone creation optional. It is disabled by default.

### CODE REVIEW

File: modules/azure/master/output.tf [hunk 1]
```diff
@@ -5,3 +5,19 @@ output "ip_address" {
 output "console_ip_address" {
   value = "${azurerm_public_ip.tectonic_console_ip.ip_address}"
 }
+
+output "ingress_external_fqdn" {
+  value = ["${azurerm_public_ip.tectonic_console_ip.*.fqdn}"]
```
Comment (Quentin-M) [inline modules/azure/master/output.tf]: Discussed offline, looks like `.*` is not required. The resource will exist anyways. The FQDN might be an empty string but that's fine.

**Comment by s-urbaniak** [comment]
@sozercan this seems to have a conflict with the current master changes. Do you mind to rebase? Thanks!

**Comment by sozercan** [comment]
@s-urbaniak should be good now, thanks!

File: config.tf [hunk 2]
```diff
@@ -163,6 +163,11 @@ variable "tectonic_ca_key_alg" {
   default     = "RSA"
 }
 
+variable "tectonic_create_dns_zone" {
+  description = "If set to true, create an Azure DNS zone"
```
Comment (Quentin-M) [inline config.tf]: This is an Azure-specific variable. It should not be in that configuration file but in `platforms/azure/variables.tf`. Can we fix that please?

**Pull request merged**